Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md

```diff
@@ -308,6 +305,7 @@ In this task you will import a solution that contains the Power Automate flow re
 3. If **Status** is **Off**, select **...** next to **Import Data** then select **Turn On**.
 
    > **Important:** If you receive an error message, verify that the entities and fields you created match the instructions above.

```

Comment:
I had to update the flow to remove the bc_building@odata reference.

Reply:
Hi Allison, can you provide more info here? Why did you need to remove the reference? Was it not working properly?

Reply:
I did end up getting it to work correctly, so not sure what the problem was the first time, so we can ignore this comment. 

I did just have a student have an issue where the error said: 
"There are missing dependencies. Install the following solutions before installing this one: "Active"

![image](https://user-images.githubusercontent.com/71679420/103972686-52db0700-51d2-11eb-9e61-743392e2e594.png)

So any help on that error would be appreciated.

Reply:
I would verify that they have all of the entities/tables and fields/columns named exactly right.